public/components/custom_panels/__tests__/custom_panel_view.test.tsx

```diff
@@ -111,27 +134,31 @@ describe('Panels View Component', () => {
       window.location.assign(`#/event_analytics/explorer/${savedVisId}`);
     };
 
+    const services = { toasts: { addDanger: (t) => {} } }
```

Comment:
you can use https://github.com/opensearch-project/opensearch-dashboards/blob/main/src/core/TESTING.md#core-mocks

i added a mock here https://github.com/opensearch-project/dashboards-observability/blob/a4cb6d1d33d00349449932f4679144acba963ace/test/__mocks__/coreMocks.ts#L11